app/views/moderations/mod.html.erb

```diff
@@ -136,15 +136,16 @@
         <b>Current live tags:</b> <%= @moderatable.tag_list %>
         <br /><br />
         <%= f.hidden_field :article_id, value: @moderatable.id %>
+        <%= f.hidden_field :adjustment_type, value: "removal" %>
         <%= f.text_field :tag_name, placeholder: "Tag Name" %>
         <%= f.text_area :reason_for_adjustment, placeholder: "Reason for Removal (Be super kind) - Only the reason is needed, the notification will take care of the rest." %>
         <%= f.submit "Remove Tag" %>
       <% end %>
-      <% adjustments = TagAdjustment.where(article_id: @moderatable.id, adjustment_type: "removal") %>
-      <% if adjustments.any? %>
+      <% removed_adjustments = @adjustments.filter { |a| a.adjustment_type == "removal" } if @adjustments %>
```

Comment:
this should probably be a variable in the controller's action, as `@removed_adjusments`